package.json

```diff
@@ -18,6 +18,17 @@
     "start": "node --require @babel/polyfill build/app.js",
     "coverage": "nyc report --reporter=text-lcov | coveralls"
   },
+  "nyc": {
+    "exclude": [
+      "src/database/migrations",
+      "src/database/models",
+      "src/database/seeders",
+      "coverage",
+      "src/validations/signup.js",
```

Comment:
I think you need to remove the excluded files except for database files and coverage folder only.